Base/usr/share/man/man1/ps.md

```diff
@@ -18,6 +18,7 @@ For each process, print its PID (process ID), to which TTY it belongs, and invok
 * `-a`: Consider all processes that are associated with a TTY.
 * `-A` or `-e`: Consider all processes, not just those in the current TTY.
 * `-f`: Also print for each process: UID (as resolved username), PPID (parent PID), and STATE (Runnable, Sleeping, Selecting, Reading, etc.)
+* `-o column-format`:  Specify a user-defined format, as a list of column names separated by commas or spaces. 
```

Comment:
The explanation in the commit message is a lot more descriptive of the format, and would be useful to include here. :^)